examples/distinct-object-types/distinct_object_types.bal

```diff
@@ -1,30 +1,55 @@
 import ballerina/io;
 
-// The `Person` object type that contains a string field called `name`.
-type Person distinct object {
+class Person {
     public string name;
+
+    function init(string name) {
+        self.name = name;
+    }
 };
 
-// The `Engineer` and `Manager` classes are structurally the same but introducing the
-// `distinct` keyword distinguishes them by considering them as nominal types.
-distinct class Engineer {
-    *Person;
+// `DistinctPerson` is a proper subtype of `Person`.
+distinct class DistinctPerson {
+    public string name;
 
     function init(string name) {
         self.name = name;
     }
 }
 
-distinct class Manager {
-    *Person;
+// The `SomeWhatDistinctPerson` type is a subtype of the `DistinctPerson` type
+// since it includes the `DistinctPerson` type's type IDs via inclusion.
+class SomeWhatDistinctPerson {
+    *DistinctPerson;
+    
+    public string name;
+
+    function init(string name) {
+        self.name = name;
+    }
+}
+
+// EvenMoreDistinctPerson is a proper subtype of DistinctPerson since it has an additional type ID
```

Comment:
Backticks, The `T` type form, period at the end not addressed here.

Reply:
Fixed